src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function sum(a) {
+    count++;
+
+    return (b) => {
+      if (count > 3 && count % 2 === 0) {
```

Comment:
The function should return 'Bzzz... Error!' only when it is called an even number of times. Currently, it starts returning errors after the third call, which is not in the requirements.